### PR TITLE
Add adjustments from SNEK engine

### PR DESCRIPTION
**Pull request by kleberbaum**
Due to the experiences we gained from SNEK,
we decided to update our wagtail template.

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is some unexpected behavior.

**What is the new behavior (if this is a feature change)?**
- Now everything is working just fine.

**Other information**:
- This is a special PR. @kleberbaum

### CODE REVIEW

**Review by schettn — APPROVED** [review]
Lgtm

File: esite/core/wagtail_hooks.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+# @hooks.register("insert_editor_js", order=100)
+# def global_admin_js():
+#    # Add /static/css/custom.css to admin.
+#    return format_html('<script src="{}"></script>', static("wagtailadmin/page-editor.js"))
+
+# @hooks.register('construct_main_menu')
+# def hide_snippets_menu_item(request, menu_items):
+#    menu_items[:] = [item for item in menu_items if item.name != 'reports']
```
Comment (schettn) [inline esite/core/wagtail_hooks.py]:
```suggestion
```

File: esite/user/models.py [hunk 2]
```diff
@@ -1,49 +1,62 @@
...
         error_messages={"unique": "A user with that username already exists."},
         help_text="Required. 36 characters or fewer. Letters, digits and @/./+/-/_ only.",
         max_length=36,
         unique=True,
         validators=[django.contrib.auth.validators.UnicodeUsernameValidator()],
-        verbose_name="username",
     )
-    is_customer = models.BooleanField(blank=False, default=False)
-    registration_data = models.TextField(null=True, blank=False)
+    # is_enterprise = models.BooleanField("enterprise", blank=False, default=False)
```
Comment (schettn) [inline esite/user/models.py]:
```suggestion
```

File: esite/user/models.py [hunk 3]
```diff
@@ -58,17 +71,62 @@ def save(self, *args, **kwargs):
...
-        FieldPanel("is_customer"),
-        FieldPanel("registration_data"),
+        FieldPanel("first_name"),
+        FieldPanel("last_name"),
+        FieldPanel("email"),
+        FieldPanel("is_staff"),
+        FieldPanel("is_active"),
+        # FieldPanel("is_enterprise"),
+        # FieldPanel("sources"),
+        # FieldPanel("cache"),
```
Comment (schettn) [inline esite/user/models.py]:
```suggestion
```

File: esite/user/models.py [hunk 4]
```diff
@@ -58,17 +71,62 @@ def save(self, *args, **kwargs):
...
+        GraphQLString("last_name"),
+        GraphQLString("email"),
+        GraphQLBoolean("is_active"),
+        # GraphQLBoolean("is_enterprise"),
+        # GraphQLString("sources"),
+        # GraphQLString("cache"),
+        # GraphQLCollection(GraphQLForeignKey, "person", "people.Person"),
+        # GraphQLCollection(GraphQLForeignKey, "enterprise", "enterprises.Enterprise"),
+        # GraphQLCollection(GraphQLForeignKey, "talk_owner", "talk.Talk"),
+        # GraphQLCollection(GraphQLForeignKey, "comment_owner", "comment.Comment"),
```
Comment (schettn) [inline esite/user/models.py]:
```suggestion
```

File: esite/user/models.py [hunk 5]
```diff
@@ -58,17 +71,62 @@ def save(self, *args, **kwargs):
...
+#                 BasePage.content_panels + main_content_panels, heading="Content"
+#             ),
+#             ObjectList(
+#                 BasePage.promote_panels + BasePage.settings_panels,
+#                 heading="Settings",
+#                 classname="settings",
+#             ),
+#         ]
+#     )
+
```
Comment (schettn) [inline esite/user/models.py]:
```suggestion

```

File: esite/user/schema.py [hunk 6]
```diff
@@ -44,13 +38,42 @@ class Mutation(graphene.ObjectType):
...
+        registry.models[get_user_model()], token=graphene.String(required=False)
+    )
+    # user = graphene.Field(
+    #     registry.models[get_user_model()],
+    #     username=graphene.String(required=False),
+    #     token=graphene.String(required=False),
+    # )
+    # users = graphene.List(
+    #     registry.models[get_user_model()], token=graphene.String(required=False)
+    # )
```
Comment (schettn) [inline esite/user/schema.py]:
```suggestion
```

**Review by schettn — CHANGES_REQUESTED** [review]
We should remove this

**Review by kleberbaum — COMMENTED** [review]
Should be ok to merge.

**Pull request merged**